modules/ROOT/pages/introduction/intro-config-use-studio.adoc

```diff
@@ -0,0 +1,160 @@
+* The section section shows the default values for the base path and listener interceptors to use in the connection.
++
+. Accept the defaults on the *General* tab or specify required information:
+* To listen on a specific port for every network interface on the server, leave the default value `0.0.0.0` for the *Host* property.
+* To listen for incoming requests generated inside the server, set the *Port* value to `localhost`.
+. To use TLS, click the *TLS* tab and specify the required information.
+. On the *Advanced* tab, specify a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+. Set the *Path* field to the appropriate path.
+. Select the other tabs and specify the appropriate field values.
+
+For more information about HTTP Listener, see the xref:http/http-documentation.adoc[HTTP Connector Reference].
+
+=== Use Scheduler as an Input Source
+
+Use Scheduler to initiated a flow when a time-based condition is met. You can configure Scheduler to initiate a flow at a regular interval or you can use a more flexible cron expression. For example, you can use Scheduler to initiate a flow every five seconds, or you can set up a cron expression to initiate a flow at noon everyday for one month.
+
+To use Scheduler as an input source:
+
+. In the Mule Palette view, search for and select *Scheduler*:
++
+.Select Scheduler to schedule when a flow is initiated
+image::introduction/intro-config-studio-select-scheduler.png[Selecting Scheduler as an input source]
++
+. Drag *Scheduler* onto the canvas.
++
+The following screenshot shows the default values for the scheduling strategy:
++
+.Default values for the scheduling strategy
+image::introduction/intro-config-studio-scheduler.png[Default values for the scheduling strategy]
++
+. In the *Scheduling Strategy* field of the *Scheduler configuration* screen, select whether you want to use a fixed frequency or cron expression to initiate the flow.
+* If you select *Fixed Frequency*, enter values for these fields:
++
+** *Frequency*
++
+The frequency at which the Scheduler initiates the flow, based on the time unit specified in the *Time unit* field
++
+** *Start delay*
++
+The amount of time to wait before initiating the flow for the first time after the app starts, based on the time unit specified in the *Time unit* field
++
+** *Time unit*
++
+The time unit for the values of *Frequency* and *Start Delay*
++
+* If you select *Cron*, enter the cron expression in the *Expression* field and the time zone in the *Time Zone* field. For
+ the format and valid values for the *Time Zone* field, see the Javadoc for the Java TimeZone class.
++
+. To configure metadata for the payload, variables, and attributes, select the *Metadata* tab and add the appropriate metadata.
+
+For more information about Scheduler, see xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint (Trigger)].
+
+=== Use a Connector-Specific Input Source
```

Comment:
I don't think this should be in an intro topic. I think the only thing that should be in the intro is conceptual information. Procedures should be with the connector docs.

Reply:
Again, I am trying to document a general path for configuring a connector. When the user guides are beefed up, we can consider removing this.